gradle/versions.gradle

```
@@ -128,6 +128,10 @@ dependencyManagement {
       entry 'mockito-junit-jupiter'
     }
 
+    dependencySet(group: 'io.github.hakky54', version: '2.7.10') {
+      entry 'logcaptor'
+    }
```

Given there's only a single entry here we can probably just use the single line version of this:
```suggestion
    dependency 'io.github.hakky54:logcaptor:2.7.10'
```